pulsar-client/src/main/java/org/apache/pulsar/client/api/Consumer.java

```diff
@@ -278,4 +277,19 @@
      * @return a future to track the completion of the seek operation
      */
     CompletableFuture<Void> seekAsync(MessageId messageId);
+
+    /**
+     * Gets last message id of Topic.
```

Comment:
Be more explicit about what the last message id of the topic actually is. It's the message id of the last message for the topic which has been published successfully to the broker.

Reply:
Thanks. will add it.